### PR TITLE
Set the scheduler options in the builder

### PR DESCRIPTION
**Pull request by BECATRUE**
I added the scheduler options in the builder app.

### Functions
There are three scheduler options.
- Pipeline: The pipeline to run the experiment in.
- Priority: Higher value means sooner scheduling.
- Timed: The due date for the experiment in ISO format. None for no due date.

I added these options using several `Entry` classes implemented in #47.

### Examples
image
image

### CODE REVIEW

**Comment by BECATRUE** [comment]
I will implement the unittest for this issue in the next PR, so please ignore the unittest result.

Also, I modify the approving mechanism that at lease one of Jiyong and I should approve and the number of approving reviews is at least 2, but I'm not sure if it works well...

File: iquip/apps/builder.py
Comment (kangz12345) [inline iquip/apps/builder.py]: You can use "according to" or "in accordance with".

File: iquip/apps/builder.py
Comment (kangz12345) [inline iquip/apps/builder.py]: Considering the GUI, how about checking `checkBox` instead of `dateTimeEdit`?
Not going to happen but `dateTimeEdit` can be enabled or disabled via codes and the users will see the checkbox to know if the "timed" option is enabled.

Comment (kangz12345) [inline iquip/apps/builder.py]: Please avoid using the backslash.

File: iquip/apps/builder.py
Comment (kangz12345) [inline iquip/apps/builder.py:245]: Please write the attribute docstring for `schedOptsListWidget`.

File: iquip/apps/builder.py
Comment (kangz12345) [inline iquip/apps/builder.py]: You can use [`dict.update()`](https://docs.python.org/3/library/stdtypes.html#dict.update): `params.update(self.schedOpts)`

File: iquip/apps/builder.py
Comment (kangz12345) [inline iquip/apps/builder.py]: Gets ... and returns them.

**Review by kangz12345 — CHANGES_REQUESTED** [review]

File: iquip/apps/builder.py
Comment (BECATRUE) [reply iquip/apps/builder.py]: Then, do you mean using a parentheses?
I think we should make an explicit line continuation using a backslash or a parentheses.

Comment (BECATRUE) [reply iquip/apps/builder.py]: Oh, it looks more clear! I will apply it.

**Comment by BECATRUE** [comment]
Umm.. I misused the rebase option, so I think the CODEOWNER file has been added.
But.. it seems not to make a problem, so please ignore it😅

File: iquip/apps/builder.py
Comment (kangz12345) [reply iquip/apps/builder.py]: If the line becomes too long, you have to separate the code into multiple lines.
For example, you should not use the if-else _expression_ (or, the ternary operator) in that case. You should rather use the if-else _statement_.

Backslashes are not that readable (although this case was quite simple) in general.
If you have to write a long line, parentheses are preferred as stated in [PEP8](https://peps.python.org/pep-0008/#maximum-line-length).

File: iquip/apps/builder.py
Comment (kangz12345) [inline iquip/apps/builder.py]: returns!

File: iquip/apps/builder.py
Comment (kangz12345) [inline iquip/apps/builder.py]: I think just using the `if-else` statement would look better!
```python
if self.checkBox.isChecked():
    return self.dateTimeEdit.dateTime().toString(Qt.ISODate)
return None
```
Or is there any reason you want to use the ternary operator?

**Review by kangz12345 — CHANGES_REQUESTED** [review]

**Comment by kangz12345** [comment]
Will #68 resolve the unit test errors?

**Comment by BECATRUE** [comment]
> Will #68 resolve the unit test errors?

Yes! the error will be handled in PR #70 of the issue #68.

File: iquip/apps/builder.py
Comment (BECATRUE) [reply iquip/apps/builder.py]: Oh, I couldn't comp up with an idea to use `if-else` statement😅
I will apply it.

**Review by kangz12345 — APPROVED** [review]
LGTM

**Pull request merged**
